### PR TITLE
Add asset elements into README.md and into the code samples unit tests

### LINKED ISSUE #144: Add support for elements in Asset entity
### Motivation

The feature "asset tagging", internally represented as the presence of Taxonomy elements in the Asset entity was introduced to the UI a while ago and now has made its way to the MAPI as well. Now is the time to add support to MAPI SDKs.

### Proposed solution

Introduce new property `Elements` to the Asset entity and make it work. The idea for the API to be exactly the same as elements in language variants, although Asset currently only supports Taxonomy elements that are non-localizable. The API and SDKs, however, should be "generic" enough so that any type element can be easily supported in the future.

### Additional context

The feature will be released ~7.3.2022. Docs are not public yet.

The `Elements` property is being returned from the GET endpoints and is optionally accepted in the POST/PUT endpoints. Only elements that are present during insert/update/upsert will be modified.

### PR DESCRIPTION
**Pull request by Simcsa**
### Motivation

Fixes #144.

New **Working with assets** section added to the readme. The section contains two examples of updating asset elements - strongly-typed and dynamic.

Also, added asset elements into the code samples unit tests.

### Checklist

- [ ] Code follows coding conventions held in this repo
- [x] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

No need to test

### CODE REVIEW

**Comment by Suzii** [comment]
Shouldn't this PR be pointed at vNext branch?

**Comment by gormal** [comment]
> Shouldn't this PR be pointed at vNext branch?
Yes, this change belongs to vNext branch, however the readme.md is quite different there. I would suggest adding a new section "Working with assets" that would contain info about dynamic assets vs strongly typed assets and a simple code sample on how to create assets in both categories.

@Simply007 what do you think?

**Comment by Simply007** [comment]
There is also a https://github.com/Kentico/kontent-management-sdk-net/blob/vNext/Kentico.Kontent.Management.Tests/Unit/CodeSamples/Readme.cs file with all existing code samples for  README to just verify the code is compilable.

**Comment by Simcsa** [comment]
Okay, I added "Working with assets" section with two examples. The PR is now pointed to the vNext branch. The unit tests were updated.

**Review by gormal — CHANGES_REQUESTED** [review]
Looks great to me, there is just one comment.

File: Kentico.Kontent.Management.Tests/Unit/CodeSamples/Readme.cs [hunk 1]
```diff
@@ -264,5 +274,82 @@ public async void QuickStartAddLanguageVariant()
...
+                FileReference = fileResult,
+                Elements = taxonomyElements
+            };
+
+            string externalId = "ext-hello-world";
+
+            // Remove next line in codesample
+            client = _fileSystemFixture.CreateMockClientWithResponse("AssetResponse.json");
+            // Upserts an asset by external ID
+            var response = await client.UpsertAssetByExternalIdAsync(externalId, asset);
```
[the remark this answers is not in the document]
Comment (Simcsa) [reply Kentico.Kontent.Management.Tests/Unit/CodeSamples/Readme.cs]: UpdateAssetAsync used

**Pull request merged**